packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -1847,6 +1937,55 @@ function applyOptimisticUpdates_forRoomSubscriptionSettings(
   return roomSubscriptionSettingsByRoomId;
 }
 
+/**
+ * Applies optimistic updates to subscriptions in a stable way.
+ */
+function applyOptimisticUpdates_forSubscriptions(
```

Comment:
I left some `TODO`s for non-implemented optimistic updates that I think we could stay without given how non-critical they are.